nym-vpn-cli/src/main.rs

```diff
@@ -124,6 +128,22 @@ async fn run_vpn(args: commands::RunArgs) -> Result<()> {
     Ok(())
 }
 
+async fn import_credential(
+    args: commands::ImportCredentialArgs,
+    config_path: Option<PathBuf>,
+) -> Result<()> {
+    let config_path = config_path.expect("config path not set");
```

Comment:
Seems like no reason to use it as `Option` here

Reply:
Yes you're right, I'll add proper error handling here